confirmation/models.py

```diff
@@ -107,7 +108,7 @@ def send_confirmation(self, obj, email_address, additional_context=None,
         else:
             template_name = obj._meta.model_name
         templates = [
-            'confirmation/%s_confirmation_email_subject.txt' % (template_name,),
+            'confirmation/%s_confirmation_email.subject' % (template_name,),
             'confirmation/confirmation_email_subject.txt',
```

Comment:
Alright, I was acutally wondreing what the purpose of this file is, since it doesn't exist. Do you have any idea?